pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -774,17 +775,9 @@ LogicalResult ConvertAtenOp<AtenGeluOp>::matchAndRewrite(
   return success();
 }
 
-using ReductionConvFunc = llvm::Optional<Value> (*)(
-    PatternRewriter&,
-    Operation*,
-    RankedTensorType,
-    Value,
-    ElementsAttr,
-    bool);
-
 // They all constitute a common form invoking the appropriate
 // converion function in MhloLegalizeCommon.cpp
-template <typename AtenOpT, ReductionConvFunc ConversionFuncT>
+template <typename AtenOpT, typename MhloOpT, bool isMean>
```

Comment:
how about make `isMean` a constexpr like `std::is_same<MhloOpt, mhlo::MeanOp>` to remove this template arugment?